spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala

```diff
@@ -48,55 +77,81 @@ class MahoutSparkILoop extends SparkILoop {
 
     conf.set("spark.executor.memory", "1g")
 
-    sparkContext = mahoutSparkContext(
+    _interp.sparkContext = mahoutSparkContext(
       masterUrl = master,
       appName = "Mahout Spark Shell",
       customJars = jars,
       sparkConf = conf
     )
 
```

Comment:
uggh.. of course-- i was doing it backwards.  the SparkContext (along with the SparkDistributedContext) have to be created by a call to ```createSparkContext()``` before we can access sdc (hence the above NPE).
```
   _interp.interpret("""
         @transient val sc = {
           val _sc = org.apache.spark.repl.Main.interp.createSparkContext()
           println("Spark context available as sc.")
           _sc
         }
              """)

      _interp.interpret("""

         @transient implicit val sdc: org.apache.mahout.sparkbindings.SparkDistributedContext =
            org.apache.spark.repl.Main.interp.asInstanceOf[org.apache.mahout.sparkbindings.shell.MahoutSparkILoop].sdc

                        """)

      echoToShell("Mahout distributed context is available as \"implicit val sdc\".")
```

works fine.  Had it in my head that  2 SparkContexts were being created rather than 2 MahoutSparkContextes...